core/src/net/sf/openrocket/rocketcomponent/InnerTube.java

```diff
@@ -177,35 +178,51 @@ public void setInstanceCount( final int newCount ){
 	 * pack inside each other.
 	 */
 	public double getClusterScale() {
-		return clusterScale;
+		return getClusterSeparation() / (getOuterRadius() * 2);
+	}
+
+	/**
+	 * Set the cluster scale.
+	 */
+	public void setClusterScale(double scale) {
+		double separation = scale * getOuterRadius() * 2;
+		setClusterSeparation(separation);
 	}
 	
 	@Override
 	public boolean isAfter(){ 
 		return false;
 	}
-	
+
 	/**
-	 * Set the cluster scaling.
-	 * @see #getClusterScale()
+	 * Return the distance between the closest two cluster inner tube center points.
+	 * This is equivalent to the cluster scale multiplied by the tube diameter.
```

Comment:
I would remove this line, since we're gonna remove all references to the cluster scale after we remove backward compatibility with OR 15.03.

Reply:
Removed